_posts/2021-04-19-elixir-is.md

```diff
@@ -0,0 +1,16 @@
+---
+site: [freiburg]
```

Comment:
@bgruening all sites?

Reply:
why not and maybe the title as Galaxy community?